packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -444,6 +483,23 @@ export abstract class SharedObjectCore<TEvent extends ISharedObjectEvents = ISha
      * when the op is ACKed or resubmitted, respectively
      */
     protected abstract applyStashedOp(content: any): unknown;
+
+    public emit(event: EventEmitterEventType, ...args: any[]): boolean {
```

Comment:
can you add a comment and mark this internal, as only the dds themselves should call this method